evaluation.md

```diff
@@ -0,0 +1,413 @@
+
+# TAAR – Evaluating existing recommenders
+
+Not every recommender can always make a recommendation. To evaluate the individual recommenders for the ensemble, we want to find out how often this is the case and how well the recommenders complement each other.
+
+This notebook either needs to be executed in the [TAAR](http://github.com/mozilla/taar) repository or somewhere where TAAR is in the Python path, because some TAAR recommenders are loaded in.
+
+## Retrieving the relevant variables from the longitudinal dataset
+
+
+```python
+frame = sqlContext.sql("""
+SELECT client_id,
+    settings[1].locale AS locale,
+    map_keys(active_addons[1]) AS installed_addons,
+    geo_city[1] AS geoCity,
+    subsession_length[1] AS subsessionLength,
+    system_os[1].name AS os,
+    scalar_parent_browser_engagement_total_uri_count[1].value AS total_uri,
+    scalar_parent_browser_engagement_tab_open_event_count[1].value as tab_open_count,
+    places_bookmarks_count[1].sum as bookmark_count,
+    scalar_parent_browser_engagement_unique_domains_count[1].value as unique_tlds
+FROM longitudinal
```

Comment:
May want to add a filter to ensure only valid Fx clients end up in here... there is a lot of wierdness in telemetry.

`WHERE normalized_channel='release'
          AND build IS NOT NULL
          AND build[0].application_name='Firefox'`

could do the trick.

Reply:
This does not appear to have been addressedin the latest commit. Please double check that only valid FX clients are being included.

Reply:
Sorry for that. I added it in the beginning but forgot to re-add it when changing the query for Hive.